### ISSUE TITLE
Copy system info to clipboard

### ISSUE THREAD

**Issue by peteruithoven** [elementary/switchboard-plug-about#47]
When reporting issues it's usually important to include system information. It would be quite useful if we could include a button that would copy "relevant" system info in text only form to the clipboard. 
Maybe it's even possible to add this as an action to the Applications menu, like "Screenshot of a Selected area"
An example of what that content might be is what neofetch outputs:
```
$ neofetch
         eeeeeeeeeeeeeeeee            peteruithoven@xps
      eeeeeeeeeeeeeeeeeeeeeee         -----------------
    eeeee  eeeeeeeeeeee   eeeee       OS: elementary OS 0.4.1 Loki x86_64
  eeee   eeeee       eee     eeee     Model: XPS 15 9550
 eeee   eeee          eee     eeee    Kernel: 4.4.0-112-generic
eee    eee            eee       eee   Uptime: 16 hours, 25 mins
eee   eee            eee        eee   Packages: 3098
ee    eee           eeee       eeee   Shell: bash 4.3.48
ee    eee         eeeee      eeeeee   Resolution: 3840x2160
ee    eee       eeeee      eeeee ee   DE: Pantheon
eee   eeee   eeeeee      eeeee  eee   WM: Mutter(Gala)
eee    eeeeeeeeee     eeeeee    eee   Terminal: pantheon-termin
 eeeeeeeeeeeeeeeeeeeeeeee    eeeee    Terminal Font: Roboto Mono 10
  eeeeeeee eeeeeeeeeeee      eeee     CPU: Intel i7-6700HQ (8) @ 3.500GHz
    eeeee                 eeeee       GPU: NVIDIA GeForce GTX 960M
      eeeeeee         eeeeeee         GPU: Intel HD Graphics 530
         eeeeeeeeeeeeeeeee            Memory: 7444MiB / 32038MiB
```

My personal suggestion:
```
OS: elementary OS 0.4.1 Loki (Built on "Ubuntu 16.04.3 LTS")
Kernel: 4.4.0-112-generic
Widget toolkit: GTK+ 3.18.9
Desktop environment: Pantheon
Window manager: Mutter (Gala)
Shell: bash 4.3.48
Terminal: pantheon-termin
CPU: Intel i7-6700HQ (8) @ 3.500GHz
GPU: NVIDIA GeForce GTX 960M
Memory: 32 GB
Storage: 856,1 GB
```

**Comment by ssspacez** [comment]
I agree, a "Copy to clipboard" button would be nice, it's useful when you are asking for support or just want to share you hardware info.

**Comment by KeironO** [comment]
I don't think this is a bad idea, but I'm not sure how to go about implementing it design wise.

I've got something working, but it looks like this right now.

!(Image of Adjustments)[https://i.imgur.com/2FwsEOD.png]

It would be nice to have a bit of input as to whether or not this is the right way of going about doing this.

**Comment by peteruithoven** [comment]
Good question... I think it's a bit weird to place the button on one side, since it contains info from both sides. 
But we probably also don't want to add another large button to the list of buttons on the bottom, because that's getting cluttered. 
Maybe we could use a button with a copy / clipboard icon? Like `edit-copy`?
![screenshot from 2019-01-21 00 09 12](https://user-images.githubusercontent.com/523210/51446356-41f90880-1d11-11e9-937f-3eeb652c8275.png)
Maybe place that button in the center on the line or centered below the line?

**Comment by KeironO** [comment]
Are there any other elementary applications that do this? I'm trying to keep within the remit of the design style.

Thanks.

**Comment by naaando** [comment]
What if a flat button is used, it would look like a link, and I don't think placing it at one side should be a issue, people probably will try before assuming something else I guess.

**Comment by aquaductape** [comment]
What if the text when hovered, a "Copy to Clipboard" popover is shown, that way when clicked, it is stored to the clipboard. This prevents the About window to be cluttered with extra buttons. Downside is that it's not intuitive and user only finds out when hovering.

**Comment by jeremypw** [comment]
Only info needed by the elementary issue templates should be copied, not anything irrelevant or with security/privacy implications.

**Comment by jeremypw** [comment]
Tbh I think the row of buttons along the bottom looks ugly anyway.  Maybe an MenuButton on the headerbar could subsitute for these and contain the Copy to Clipboard?  That would be outside the scope of this issue though.

**Comment by ChildishGiant** [comment]
Now that system details are split over 3 tabs this would be more useful than ever.
